lib/ansible/plugins/action/__init__.py

```diff
@@ -1338,9 +1330,6 @@ def _low_level_execute_command(self, cmd, sudoable=True, in_data=None, executabl
         if rc is None:
             rc = 0
 
-        # be sure to remove the BECOME-SUCCESS message now
-        out = self._strip_success_message(out)
```

Comment:
this should really happen in the connection

Reply:
It is in some but seems like not all which is the problem.

Reply:
probably why i left it, so deprecate it instead (or move to connection/become base class?)

Reply:
The trouble is there is really no way to deprecate this functionality without potentially breaking things in the connection plugins that don't clean it up. We can fix up our connection plugins but it could still break any connection plugins outside of Ansible.

Reply:
that is why we deprecate, give them a message for 4 versions and then remove (then it will break, but they were warned)

Reply:
I suppose we do detect if the message is in there and add a warning to tell it to fix up their code. Makes sense.